db/migrate/20190821082807_add_defaults_for_data_provider_roles.rb

```diff
@@ -1,11 +1,12 @@
 class AddDefaultsForDataProviderRoles < ActiveRecord::Migration[5.2]
   def change
-    DataProvider.all.each do |data_provider|
-      data_provider.role_point_of_interest = true
-      data_provider.role_tour = true
-      data_provider.role_news_item = true
-      data_provider.role_event_record = true
-      data_provider.save
-    end
+    # This migration is no longer needed.
```

Comment:
why are they not needed anymore?

Reply:
@marcometz told me that

Reply:
😆 ok

Reply:
beim init setup (freelancer) gibt es keine data_provider deren daten aktualisiert werden müssen.

Reply:
It is not possible to delete the migration, otherwise there would be inconsistencies in the schemas table.